tests/flow/utils.py

```diff
@@ -0,0 +1,66 @@
+from RLTest import Env
```

Comment:
I think it would be nicer if we will rename this file to multiproc.py and place it within /tests/flow/utils/multiproc.py